mockData/mockData.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "merchant": "Safeway",
```

Comment:
Why is this field necessary? We don't have any display for merchant information in our design spec